_articles/the-three-factors-of-authentication.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: article
+title: "The three factors of authentication"
+abstract: "All methods of authentication, belong to one of three types; what you know, what you have, and what your are."
+share_image: /assets/images/key-business-concept-with-flat-design/cropped.jpg
+sitemap:
+  lastmod: 2020-03-124
+---
+
+![](/assets/images/key-business-concept-with-flat-design/cropped.jpg)
+
+## What is authentication
+
+Authentication is the process of validating the identity of a user to a system.
+There are many different ways a user can demonstrate their identity and so be authenticated,
+They methods range from presenting a train ticket or passport in the physical world, to remembering a password for an online account.
+
+All methods of authentication can be grouped into one of three types.
+
+## What you know
+
+The simplest example of the "what you know" factor is a password, it can also be called a pin, pass phrase or security question.
+These are known as **The knowledge factors**.
+
+Knowledge of the secret, is used to confirm your identity.
+
+The strength of this type of authentication method lies in it's simplicity.
+Once a user knows a password they are able to reveal it where ever it is required to be authenticated.
+
+Another strength of this method is that there is nothing to be lost or stolen.
+Assuming the password has not been written down or stored anywhere.
+
+A weakness of this approach is the difficulty in restricting knowledge of the secret to a single individual.
+It is important to realise, a user must reveal their password to a system to authenticate.
+Once the user has done that, the system also knows the secret and can use it to impersonate the user.
+
+This problem with passwords is the reason you are required to use a different passwords for each service that uses passwords.
+
+Another problems is forgetfulness.
+Human beings are not perfect at retaining information, and so there is a limit to the number of secrets someone can reliably remember.
+
+## What you have
+
+A train ticket is an example of possessing something to prove authenticate a user.
```

Comment:
example of something physical you have that can be used to authenticate you to a system such as the train conductor.